components/script/dom/subtlecrypto.rs

```diff
@@ -1520,7 +1856,11 @@ impl SubtleCrypto {
             128 => Handle::Aes128(data.to_vec()),
             192 => Handle::Aes192(data.to_vec()),
             256 => Handle::Aes256(data.to_vec()),
-            _ => return Err(Error::Data),
+            _ => {
+                println!("{}", String::from_utf8_lossy(data));
+                println!("Bad data length: {}", data.len());
```

Comment:
I think these are debug statements you forgot to remove?

Reply:
Yes they are, my bad